core/tauri/Cargo.toml

```
@@ -87,7 +87,7 @@ ico = { version = "0.2.0", optional = true }
 encoding_rs = "0.8.31"
 
 [target."cfg(any(target_os = \"macos\", windows, target_os = \"linux\", target_os = \"dragonfly\", target_os = \"freebsd\", target_os = \"openbsd\", target_os = \"netbsd\"))".dependencies]
-rfd = { git = "https://github.com/PolyMeilex/rfd", version = "0.10", optional = true }
+rfd = { version = "0.10.1", optional = true }
```

version `0.10.1` has been yanked yesterday: https://crates.io/crates/rfd/versions
```suggestion
rfd = { version = "0.11.0", optional = true }
```